demisto_sdk/commands/common/tests/integration_test.py

```diff
@@ -2181,7 +2181,7 @@ def test_additional_info_contained(self):
     }
     HIDDEN_TRUE_BUT_REPLACED_USER = {
         "configuration": [
-            {"type": 4, "display": "Username", "hidden": True},
+            {"type": 0, "display": "Username", "hidden": True},
```

Comment:
please check all the allowed, as well as one that isn't allowed (which should be considered erroneous)